### YamlMime:ManagedReference
items:
- uid: System.Xaml.XamlSchemaException
  id: XamlSchemaException
  children:
  - System.Xaml.XamlSchemaException.#ctor
  - System.Xaml.XamlSchemaException.#ctor(System.String)
  - System.Xaml.XamlSchemaException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  - System.Xaml.XamlSchemaException.#ctor(System.String,System.Exception)
  langs:
  - csharp
  name: XamlSchemaException
  nameWithType: XamlSchemaException
  fullName: System.Xaml.XamlSchemaException
  type: Class
  summary: "A kivétellel, hogy amikor egy kötés vagy egy másik séma ábrázolását és XAML jelzi a séma környezetében kivételt vált ki."
  remarks: "A XamlSchemaException fordul elő a CLR-beli kötés támogatási osztályok, amelyek a séma környezetében .NET-keretrendszer XAML-szolgáltatások számos esetben által (<xref:System.Xaml.XamlXmlReader>, <xref:System.Xaml.XamlObjectWriter>, vagy <xref:System.Xaml.XamlServices>).</xref:System.Xaml.XamlServices> </xref:System.Xaml.XamlObjectWriter> </xref:System.Xaml.XamlXmlReader> Egy XamlSchemaException általában átadása más kivételeket, a belső kivétel üzenetében, például a <xref:System.Xaml.XamlMember.IsUnknown%2A?displayProperty=fullName>-tulajdonsága true egy <xref:System.Xaml.XamlObjectWriterException>, ezért közvetlenül tényleges egy XamlSchemaException, kivéve, ha egy összetevő, például egy speciális objektum író webkiszolgálókból ritka kell lennie.</xref:System.Xaml.XamlObjectWriterException> </xref:System.Xaml.XamlMember.IsUnknown%2A?displayProperty=fullName>"
  syntax:
    content: 'public class XamlSchemaException : System.Xaml.XamlException'
  inheritance:
  - System.Exception
  - System.Xaml.XamlException
  implements: []
  inheritedMembers:
  - System.Xaml.XamlException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  - System.Xaml.XamlException.LineNumber
  - System.Xaml.XamlException.LinePosition
  - System.Xaml.XamlException.Message
  platform:
  - net462
- uid: System.Xaml.XamlSchemaException.#ctor
  id: '#ctor'
  parent: System.Xaml.XamlSchemaException
  langs:
  - csharp
  name: XamlSchemaException()
  nameWithType: XamlSchemaException.XamlSchemaException()
  fullName: System.Xaml.XamlSchemaException.XamlSchemaException()
  type: Constructor
  assemblies:
  - System.Xaml
  namespace: System.Xaml
  summary: "Egy új példányát inicializálja a <xref href=&quot;System.Xaml.XamlSchemaException&quot;> </xref> osztály rendszer által megadott üzenetet, amely leírja a hibát."
  syntax:
    content: public XamlSchemaException ();
    parameters: []
  overload: System.Xaml.XamlSchemaException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Xaml.XamlSchemaException.#ctor(System.String)
  id: '#ctor(System.String)'
  parent: System.Xaml.XamlSchemaException
  langs:
  - csharp
  name: XamlSchemaException(String)
  nameWithType: XamlSchemaException.XamlSchemaException(String)
  fullName: System.Xaml.XamlSchemaException.XamlSchemaException(String)
  type: Constructor
  assemblies:
  - System.Xaml
  namespace: System.Xaml
  summary: "Egy új példányát inicializálja a <xref href=&quot;System.Xaml.XamlSchemaException&quot;> </xref> osztály egy adott üzenet, amely leírja a hibát."
  syntax:
    content: public XamlSchemaException (string message);
    parameters:
    - id: message
      type: System.String
      description: "A kivétel leíró üzenet. Ez a konstruktor védőfalkezelőbe annak érdekében szükség, hogy ez a karakterlánc a jelenlegi rendszer kulturális környezethez tartozó honosított."
  overload: System.Xaml.XamlSchemaException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Xaml.XamlSchemaException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  id: '#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)'
  parent: System.Xaml.XamlSchemaException
  langs:
  - csharp
  name: XamlSchemaException(SerializationInfo,StreamingContext)
  nameWithType: XamlSchemaException.XamlSchemaException(SerializationInfo,StreamingContext)
  fullName: System.Xaml.XamlSchemaException.XamlSchemaException(SerializationInfo,StreamingContext)
  type: Constructor
  assemblies:
  - System.Xaml
  namespace: System.Xaml
  summary: "Egy új példányát inicializálja a <xref href=&quot;System.Xaml.XamlSchemaException&quot;> </xref> a szerializált adatokban osztályt."
  syntax:
    content: protected XamlSchemaException (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context);
    parameters:
    - id: info
      type: System.Runtime.Serialization.SerializationInfo
      description: "A szerializált objektum adatokat tartalmazó objektum."
    - id: context
      type: System.Runtime.Serialization.StreamingContext
      description: "A cél- és környezeti információt."
  overload: System.Xaml.XamlSchemaException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Xaml.XamlSchemaException.#ctor(System.String,System.Exception)
  id: '#ctor(System.String,System.Exception)'
  parent: System.Xaml.XamlSchemaException
  langs:
  - csharp
  name: XamlSchemaException(String,Exception)
  nameWithType: XamlSchemaException.XamlSchemaException(String,Exception)
  fullName: System.Xaml.XamlSchemaException.XamlSchemaException(String,Exception)
  type: Constructor
  assemblies:
  - System.Xaml
  namespace: System.Xaml
  summary: "Egy új példányát inicializálja a <xref href=&quot;System.Xaml.XamlSchemaException&quot;> </xref> megadott hibaüzenet, és hivatkozni kell a belső kivételben találhatók okozza-e ezt a kivételt, a osztályt."
  remarks: "Kivételt vált ki, mert egy előző kivétel közvetlen eredménye tartalmaznia kell egy hivatkozást az előző kivétel az a <xref:System.Exception.InnerException%2A>tulajdonság.</xref:System.Exception.InnerException%2A> A <xref:System.Exception.InnerException%2A>tulajdonság a konstruktornak átadott ugyanazt az értéket adja vissza vagy `null` Ha a <xref:System.Exception.InnerException%2A>tulajdonság nem adja meg a belső kivétel értékét a konstruktor.</xref:System.Exception.InnerException%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public XamlSchemaException (string message, Exception innerException);
    parameters:
    - id: message
      type: System.String
      description: "A kivétel leíró üzenet. Ez a konstruktor védőfalkezelőbe annak érdekében szükség, hogy ez a karakterlánc a jelenlegi rendszer kulturális környezethez tartozó honosított."
    - id: innerException
      type: System.Exception
      description: "A kivétellel, hogy a jelenlegi kivétel okozza. Ha a `innerException` paraméter nem <xref uid=&quot;langword_csharp_null&quot; name=&quot;null&quot; href=&quot;&quot;> </xref>, a jelenlegi kivétel a következik be egy <xref uid=&quot;langword_csharp_catch&quot; name=&quot;catch&quot; href=&quot;&quot;> </xref> blokk, amely kezeli a belső kivételben találhatók."
  overload: System.Xaml.XamlSchemaException.#ctor*
  exceptions: []
  platform:
  - net462
references:
- uid: System.Xaml.XamlException
  isExternal: false
  name: System.Xaml.XamlException
- uid: System.Xaml.XamlSchemaException.#ctor
  parent: System.Xaml.XamlSchemaException
  isExternal: false
  name: XamlSchemaException()
  nameWithType: XamlSchemaException.XamlSchemaException()
  fullName: System.Xaml.XamlSchemaException.XamlSchemaException()
- uid: System.Xaml.XamlSchemaException.#ctor(System.String)
  parent: System.Xaml.XamlSchemaException
  isExternal: false
  name: XamlSchemaException(String)
  nameWithType: XamlSchemaException.XamlSchemaException(String)
  fullName: System.Xaml.XamlSchemaException.XamlSchemaException(String)
- uid: System.String
  parent: System
  isExternal: true
  name: String
  nameWithType: String
  fullName: System.String
- uid: System.Xaml.XamlSchemaException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  parent: System.Xaml.XamlSchemaException
  isExternal: false
  name: XamlSchemaException(SerializationInfo,StreamingContext)
  nameWithType: XamlSchemaException.XamlSchemaException(SerializationInfo,StreamingContext)
  fullName: System.Xaml.XamlSchemaException.XamlSchemaException(SerializationInfo,StreamingContext)
- uid: System.Runtime.Serialization.SerializationInfo
  parent: System.Runtime.Serialization
  isExternal: false
  name: SerializationInfo
  nameWithType: SerializationInfo
  fullName: System.Runtime.Serialization.SerializationInfo
- uid: System.Runtime.Serialization.StreamingContext
  parent: System.Runtime.Serialization
  isExternal: true
  name: StreamingContext
  nameWithType: StreamingContext
  fullName: System.Runtime.Serialization.StreamingContext
- uid: System.Xaml.XamlSchemaException.#ctor(System.String,System.Exception)
  parent: System.Xaml.XamlSchemaException
  isExternal: false
  name: XamlSchemaException(String,Exception)
  nameWithType: XamlSchemaException.XamlSchemaException(String,Exception)
  fullName: System.Xaml.XamlSchemaException.XamlSchemaException(String,Exception)
- uid: System.Exception
  parent: System
  isExternal: true
  name: Exception
  nameWithType: Exception
  fullName: System.Exception
- uid: System.Xaml.XamlSchemaException.#ctor*
  parent: System.Xaml.XamlSchemaException
  isExternal: false
  name: XamlSchemaException
  nameWithType: XamlSchemaException.XamlSchemaException
